system/filesystem.py

```diff
@@ -41,6 +41,12 @@
     description:
     - If yes, allows to create new filesystem on devices that already has filesystem.
     required: false
+  growfs:
+    choices: [ "yes", "no" ]
+    default: "no"
+    description:
+    - If yes, if the block device and filessytem size differ, grow the filesystem into the space. This DOES NOT support shrinking. Will only grow if the FS is mounted.
```

Comment:
True, it is. I'm just thinking about the general case. It seems like a lot of extra work and potential states to deal with in a way, but I can see in the long term it's the right solution. I'll work on shrink as well then, but for now, only ext*.

Yeah, I'll add the check then for xfs, and word the warning differently.